callback_go1_15_darwin_amd64.go

```diff
@@ -0,0 +1,13 @@
+//go:build !go1.16
```

Comment:
This is just my preference, but what about the file name like

```
callback_go115_darwin_amd64.go
```

?

Reply:
Or maybe just `callback_go15_darwin_amd64.go` since go2 won't happen anytime soon?

Reply:
Hmm, `callback_go1_15_darwin_amd64.go` would be clearer than go15 (or even go115).

OK, let's keep with the current name `callback_go1_15_darwin_amd64.go`. Sorry for going back and forth.